File: examples/src/Presentations/Properties/ReadOnlyRecommended.cs
using System
using System.Collections.Generic
using System.IO
using System.Linq
using System.text
using System.Threading.Tasks
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export

/*
The example shows of using read-only recommendation for presentation (this feature was introduced in PowerPoint 2019).
When enabled it makes PowerPoint to open the presentation in read-only mode first.
*/
namespace CSharp.Presentations.properties
{
    class ReadOnlyRecommended
    {
        public static void Run()
        {
            outPptxPath = Path.Combine(RunExamples.OutPath, "ReadOnlyRecommended.pptx")

            with slides.Presentation() as pres:
            {
                pres.ProtectionManager.ReadOnlyRecommended = True
                pres.save(outPptxPath, slides.export.SaveFormat.PPTX)
            }
        }
    }
}

File: examples/src/Presentations/Properties/ReadOnlyRecommended.py
import aspose.slides as slides

def props_read_only_recommended():
    outDir = "./examples/out/"

    with slides.Presentation() as pres:
        pres.protection_manager.read_only_recommended = True
        pres.save(outDir + "props_read_only_recommended_out.pptx", slides.export.SaveFormat.PPTX)
